### ISSUE TITLE
Python Virtual Environment

### ISSUE THREAD

**Issue by mridubhatnagar** [PyLadiesDelhi/talks#20]
**Abstract**
Talk revolves around what exactly is a virtual environment in Python. Need for using a virtual environment. Overtime work is done on the same. Virtual environment, virtual environment wrapper, and now pipenv.

**About**

1. Introduction
2. Need for using virtual environment
3. Advantages
4. Demo of using
        a. Virtualenv
        b. Virtualenv wrapper
        c. pipenv
5. Shortcomings and Benefits
6. Future

**Pre-requisites**

- 
Basic Python knowledge good to have

- 
Python and pip needs to be installed 
on systems

**Expected duration**
30 min

**Level**
Beginner

**Resources**

**Speaker Bio**

I am Mridu Bhatnagar. A B.Tech Computer Science and Engineer graduate. Contributed for a while in Open Source Organization coala. Was a Google Code-In mentor for the same organization. Started working as a full-time software engineer from September-2017. Have worked with startup and MNC alike.
I do love giving back to the community and paying forward.

###### - Can be done after the talk/workshop -

[Include link to slides here](link)
https://docs.google.com/presentation/d/1fKBjOzUHzGr5DwnD606HXNtCmLjnLrKQkCTnb6se6as/edit?usp=sharing

[Include link to video here](link)

**Comment by utkarsh2102** [comment]
Hey,
Are you free somewhere around 28th  October?

**Comment by mridubhatnagar** [comment]
@utkarsh2102 yes. I am the same person who opened an issue on Introduction to API's

**Comment by utkarsh2102** [comment]
Yes, yes. I know.
Which talk would you want to give first?

**Comment by mridubhatnagar** [comment]
This one.

**Comment by mridubhatnagar** [comment]
> Yes, yes. I know.
> Which talk would you want to give first?

Hi @utkarsh2102. Any update on this.

**Comment by utkarsh2102** [comment]
Hey,
Sorry for the late reply @mridubhatnagar.
Can you take a talk on 23rd December?

**Comment by mridubhatnagar** [comment]
Hey, no problem @utkarsh2102.
23rd December, what time? 
Also, I am delivering the talk on the same topic in Pydata Delhi on Dec 22. So, delivering talk on same topic in Pyladies is fine or not?

**Comment by utkarsh2102** [comment]
Yes, it's completely fine! :D
We have a meetup from 11-4.
So should I schedule your talk then?
Also, are you on Telegram? If so, can you please share your handle?

**Comment by mridubhatnagar** [comment]
@utkarsh2102 Yes, I am there on telegram.

**Comment by utkarsh2102** [comment]
Awesome.
Could you drop me a text at "utkarsh2102"?

**Comment by cocoa1231** [comment]
Hey @mridubhatnagar 
Unfortunately due to some issues with the venue, we won't be having the meetup in December. Can I confirm your talk for Jan. 6 at 91springboard NOIDA sector 15 instead?

**Comment by mridubhatnagar** [comment]
This issue can be closed. Talk delivered in linuxchix india plus pyladies birthday meetup.

**Comment by utkarsh2102** [comment]
Hi @mridubhatnagar,  
Could you please make a PR, adding your name and details about the talk on the `README.md`?

**Comment by utkarsh2102** [comment]
I'll add a `Completed` label once done and the PR is merged.

**Comment by mridubhatnagar** [comment]
Makes sense. Btw I am too lazy to do it.

**Comment by sakshivij** [comment]
@mridubhatnagar can you raise PR.? @utkarsh2102 can we close this ?

**Comment by utkarsh2102** [comment]
Hi,
I'll close this as soon as the PR is raised.
Or better yet, just add "Closes: #20" in your PR and that should be done automatically.
